pkg/instancegroups/rollingupdate_test.go

```diff
@@ -602,6 +605,262 @@ func TestRollingUpdateTaintAllButOneNeedUpdate(t *testing.T) {
 	assertGroupInstanceCount(t, cloud, "node-1", 1)
 }
 
+func TestRollingUpdateSettingsIgnoredForMaster(t *testing.T) {
```

Comment:
I've since been reconsidering whether we need this exception. Possibly we just need to add some PodDisruptionBudgets for workloads that run on the master.